src/routing_membrane.rs

```diff
@@ -514,6 +519,26 @@ impl<F> RoutingMembrane<F> where F: Interface {
         }
     }
 
+    fn send_by_name<'a, I>(&self, peers: I, serialised_msg: Bytes) where I: Iterator<Item=&'a NameType> {
+        for peer in peers {
+            self.send(self.name_to_endpoint(peer).into_iter(), &serialised_msg);
+        }
+    }
+
+    fn name_to_endpoint(&self, name: &NameType) -> Vec<&Endpoint> {
```

Comment:
Do we need the name-to endpoint in general?  I only used explicitly the routing table or the relay_map when looking up in this direction.  I explicitly did not want to port this function.  Sorry, didnt tell you this